waku/v2/node/waku_node.nim

```diff
@@ -258,10 +265,34 @@ proc new*(T: type WakuNode,
     peerManager: PeerManager.new(switch, peerStorage),
     switch: switch,
     rng: rng,
-    enr: enr,
+    enr: populatedEnr,
     announcedAddresses: announcedAddresses
   )
 
+  if discv5Enabled:
+    var discv5Enrs: seq[enr.Record]
```

Comment:
`addBootstrapNodes` needs a mutable variable. The fn can be changed to be without side-effects in a different pr, imo